src/actions/dataActions.js

```diff
@@ -61,7 +61,7 @@ export const CRUD_DELETE_SUCCESS = 'CRUD_DELETE_SUCCESS';
 export const crudDelete = (resource, id, basePath) => ({
     type: CRUD_DELETE,
     payload: { id, basePath },
-    meta: { resource, fetch: DELETE, cancelPrevious: true },
+    meta: { resource, fetch: DELETE, cancelPrevious: false },
```

Comment:
This fixes the problem, caused by the GET_LIST call cancelling the DELETE call. This made the loading count show a bad value, and avoided the refresh of the list after a deletion.